src/mixins/filing-template-mixin.ts

```diff
@@ -405,6 +408,65 @@ export default class FilingTemplateMixin extends DateMixin {
     return filing
   }
 
+  /**
+   * Builds a restoration filing from store data. Used when saving a filing.
+   * @returns the filing body to save
+   */
+  buildRestorationFiling (): any {
+    // Build the main filing.
+    // *** TODO: update
```

Comment:
This is for future tickets to update (as they add data, etc).

Ditto for parseRestorationDraft() below.